src/js/views/globals/modal/modal_views.js

```diff
@@ -100,9 +103,21 @@ const IframeFormView = View.extend({
   template: hbs`<iframe src="/formapp/"></iframe>`,
 });
 
+
+const LoadingModalView = View.extend({
+  className: 'loading-modal__wrapper',
```

Comment:
there's a `pos--absolute` class.. should probably be `u-pos--` but /shrug

Reply:
Updated in last push.